CHANGELOG.next.asciidoc

```diff
@@ -387,6 +387,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add a switch to the driver definition on SQL module to use pretty names {pull}17378[17378]
 - Remove io.time from windows {pull}22237[22237]
 - Add interval information to `monitor` metricset in azure. {pull}22152[22152]
+- Fix cast of Session ID to an int {pull}22359[22359]
```

Comment:
Sorry I still just don't feel this is clear enough. But I will give you the +1 and you can decide to change the wording or not.
```suggestion
- Change Session ID type from int to string {pull}22359[22359]
```

Reply:
Agreed, that change is better worded.